source/kraken/worker.cpp

```diff
@@ -990,6 +1009,20 @@ void Worker::direct_path(const pbnavitia::Request& request) {
                              dp_request.clockwise());
 }
 
+void Worker::isochrone_distributed(const pbnavitia::JourneysRequest& request) {
+    auto arg = JourneysArg();
+    bool has_error = set_isochrone_distributed_args(request, arg, "isochrone_distributed");
```

Comment:
As a good practice, it's best to return `false` on an error. 
It's way more readable to have something like : 
```cpp
 if (!set_isochrone_distributed_args(....)) {
        return; // and log an error...
    }
```

Reply:
Totally agree with you! However, since `set_journeys_args` used in `set_isochrone_distributed_args` is doing the other way around, I preferred to stick with that and be consistent.

This is a problem though and has to be fixed.

Reply:
Fixed for `set_journeys_args`